### DISCUSSION TITLE
I can't launch a world on minecraft? Error code -1

### DISCUSSION THREAD

**Discussion by MilitronPrime** [microsoft/openjdk#620] [Q&A] [answered]
I use mods but I have tested them and they don't seem to be the problem. Please help.
I am really not sure what the problem is, I have tried uninstalling and reinstalling Minecraft and Java. It only seems to crash when I attempt to load into a world.

When i attempt to start a world, it says:
**The game crashed whilst exception in server tick loop
Error: com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 3 column 1 path $**
I have also attached the crash log:
[crash -1.txt](https://github.com/user-attachments/files/16383942/crash.-1.txt)

I really want to play the game, I'm just not sure how. Thanks. :)

**Comment by karianna** [comment, marked as answer]
Thank you for the report!
It looks like you are reporting that the game Minecraft crashed for you. Please try the following:

Update your graphics drivers, see [this Minecraft Help Center article](https://help.minecraft.net/hc/en-us/articles/4409137348877-Minecraft-Java-Edition-Game-Crashes-and-Performance-Issues-FAQ), section "Update your Video Card Drivers";
often Minecraft crashes are caused by outdated graphics drivers
In case you are using mods, try disabling them one by one until you find the mod causing the crash, then report the crash to the mod author
If you are using a third-party launcher, try the [official launcher](https://www.minecraft.net/download) instead
If none of the above helped, and you are still experiencing the crash with the official launcher and without any mods installed, please submit this issue over at the [Mojang bug tracker](https://bugs.mojang.com/projects/MC/summary). Please search for existing reports first; in case you do not find any, create a new report and let us know about the issue number here (e.g. MC-123456).
The Mojang team will take the first look. If an OpenJDK bug is identified, the Mojang team will contact the Microsoft Build of OpenJDK team to address the issue.
